src/pages/admin-products.js

```diff
@@ -1,7 +1,19 @@
 import React from 'react';
+import history from '../utils/history';
 
 function AdminProducts() {
-  return <h1>Admin Products</h1>;
+  const newProductsPage = () => {
```

Comment:
you can handle this in the jsx by wrapping the button in a react-router-dom <Link to="/"></Link>